plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/rest/RestCatalogTestUtils.java

```diff
@@ -27,15 +27,77 @@
 import org.apache.hadoop.fs.Path;
 import org.apache.iceberg.CatalogProperties;
 import org.apache.iceberg.catalog.Catalog;
+import org.apache.iceberg.catalog.Namespace;
+import org.apache.iceberg.catalog.TableIdentifier;
+import org.apache.iceberg.catalog.ViewCatalog;
+import org.apache.iceberg.inmemory.InMemoryCatalog;
+import org.apache.iceberg.io.FileIO;
 import org.apache.iceberg.jdbc.JdbcCatalog;
+import org.apache.iceberg.jdbc.JdbcClientPool;
+import org.apache.iceberg.view.View;
+import org.apache.iceberg.view.ViewBuilder;
 import org.assertj.core.util.Files;
 
 import java.io.File;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
 
 public final class RestCatalogTestUtils
 {
     private RestCatalogTestUtils() {}
 
+    static class TestingJdbcCatalog
```

Comment:
This is needed since the upgraded 1.5 RestSessionCatalog will check if a view exists when performing a replace. This effectively means the underlying catalog being used must implement ViewCatalog otherwise the loadView call will fail since the response cannot be parsed.
To fix this, I'm implementing a TestingJdbcCatalog which implements ViewCatalog. We'll need this anyways in the REST View Catalog implementation PR. https://github.com/trinodb/trino/pull/19818

Reply:
does it mean JdbcCatalog no longer can be used to back the REST catalog? or does it mean that JdbcCatalog no longer supports views?